common/app/views/fragments/commentBox.scala.html

```diff
@@ -24,6 +24,13 @@
     <div class="d-comment-box__onboarding">
         <h3>Welcome <span class="d-comment-box__onboarding-author"></span>, you’re about to make your first comment!</h3>
         <p>Before you post, we’d like to thank you for joining the debate - we’re glad you’ve chosen to participate and we value your opinions and experiences.</p>
+
+        <div class="d-comment-box__onboarding-username">
+            <p>Please choose your username under which you would like all your comments to show up:</p>
+            <input type="text" class="d-comment-box__onboarding-username-input" placeholder="Username: 6-20 characters, letters & numbers only, no spaces" value=""/>
```

Comment:
The username requirements seem like key information; it may be better to add them as a label outside of the input area? 

Not all screen readers read placeholder text aloud:
https://www.nngroup.com/articles/form-design-placeholders/